samples/keda-sqs/specs/package-producer.yaml

```
@@ -0,0 +1,25 @@
+include:
+- producer/producer.zip
```

You can also
```suggestion
- "producer/*"
```

And then you don't have to use zip command mentioned earlier in the doc